sdk/maps/maps-search/samples-dev/searchAddressResult.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to manipulate the response `SearchAddressResult`.
+ */
+
+import { AzureKeyCredential } from "@azure/core-auth";
+// import { DefaultAzureCredential } from "@azure/identity";
+import { MapsSearchClient, KnownSearchAddressResultType } from "@azure/maps-search";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * We use fuzzySearch in this example.
+ * But the same approach can be used in all APIs that return `SearchAddressResult`.
+ */
+async function main() {
+  /** Use subscription key authentication */
+  const subscriptionKey = process.env.MAPS_SUBSCRIPTION_KEY || "";
+  const credential = new AzureKeyCredential(subscriptionKey);
+  const client = new MapsSearchClient(credential);
+
+  /** Or use Azure AD authentication */
+  // const credential = new DefaultAzureCredential();
+  // const mapsClientId = process.env.MAPS_CLIENT_ID || "";
+  // const client = new MapsSearchClient(credential, mapsClientId);
+
+  /** Get the search results and the size of current page */
+  const { results, numResults } = await client.fuzzySearch({ query: "Starbucks" });
+
+  for (let i = 0; i < numResults; i++) {
+    console.log(`${i}th result:`);
+    /**
+     * Different result types may have different properties.
+     * See the type definition of `SearchAddressResultItem` for more details.
+     */
+    switch (results[i].type) {
+      case KnownSearchAddressResultType.POI:
```

Comment:
nit: can we add `break;` to each of the case here? If customers copy and extend the code it may not be obvious to them that the switch cases fall through.

Reply:
Sure! I would also like to add some comment to make it clear for the user like the following:
```ts
case KnownSearchAddrewwResulotType.POI:
  /** Handle the response if the type is a POI */
 break;
case ....
```

Reply:
Updated in f105d2059626aee5388d40e9360cab4843368142